### PR TITLE
Add pdf download link in html only

### PR DESCRIPTION
**Pull request by fennecJ**
You can check it in [https://fennecj.github.io/lkmpg/](https://fennecj.github.io/lkmpg/)

### CODE REVIEW

**Comment by jserv** [comment]
Append `Close #64` to git commit message, so that it can be automatically close once this pull request is merged.

File: html.cfg [hunk 1]
```diff
@@ -45,5 +45,15 @@
...
+    position:absolute;
+    top: 10px;
+    right: 50px;
+}}
+
 \begin{document}
 \EndPreamble
+
+\begin{flushright}
+\href{https://github.com/sysprog21/lkmpg/releases/download/latest/lkmpg.pdf}{Download pdf document}
```
Comment (jserv) [inline html.cfg:58]: Use uppercase for PDF. That is, `PDF`.

**Comment by fennecJ** [comment]
> Append `Close #64` to git commit message, so that it can be automatically close once this pull request is merged.

Cool, I didn’t know that before.

**Comment by jserv** [comment]
> Cool, I didn’t know that before.
> Thanks for informing me about that.

See https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue

**Pull request merged**